package.json

```diff
@@ -1,53 +1,55 @@
 {
-  "name": "strapi-plugin-react-icons",
-  "version": "0.1.4",
-  "description": "A plugin for strapi to select react icons.",
-  "strapi": {
-    "name": "react-icons",
-    "description": "A plugin for strapi to select react icons.",
-    "kind": "plugin"
-  },
-  "dependencies": {
-    "@strapi/design-system": "^1.6.3",
-    "@strapi/helper-plugin": "^4.6.0",
-    "@strapi/icons": "^1.6.6",
-    "prop-types": "^15.7.2",
-    "react-icons": "^4.8.0"
-  },
-  "devDependencies": {
-    "@strapi/typescript-utils": "^4.6.0",
-    "@types/react": "^17.0.53",
-    "@types/react-dom": "^17.0.18",
-    "@types/react-router-dom": "^5.3.3",
-    "@types/styled-components": "^5.1.26",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "react-router-dom": "^5.3.4",
-    "styled-components": "^5.3.6",
-    "typescript": "4.6.3"
-  },
-  "peerDependencies": {
-    "@strapi/strapi": "^4.9.2",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "react-router-dom": "^5.3.4",
-    "styled-components": "^5.3.6"
-  },
-  "author": {
-    "name": "Daniel Pantle"
-  },
-  "maintainers": [
-    {
-      "name": "Daniel Pantle"
-    }
-  ],
-  "engines": {
-    "node": ">=14.19.1 <=18.x.x",
-    "npm": ">=6.0.0"
-  },
-  "scripts": {
-    "develop": "tsc -p tsconfig.server.json -w",
-    "build": "tsc -p tsconfig.server.json"
-  },
-  "license": "MIT"
+	"name": "strapi-plugin-react-icons",
```

Comment:
Why do you use tabs in the .json and .ts files, but two spaces in .tsx? I think it would be more consistent to use two spaces in all files, at least I would prefer that (I know everybody has his own preferences in spacing...)

Reply:
Actually I think we should add a prettier config to have the same settings, otherwise we will always be switching back and forth or having to adapt indentations and stuf manually...